projects/core/directives/dropdown/dropdown.directive.ts

```diff
@@ -34,7 +34,7 @@ export class TuiDropdownDirective
     implements TuiDropdown, AfterViewChecked, OnDestroy
 {
     @Input('tuiDropdown')
-    set open(value: boolean) {
+    set open(value: boolean | '') {
```

Comment:
Why?

Reply:
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/12021443/163877367-d1b47caf-a8b5-4cb1-ad4f-23d2a232d0da.png">

<img width="382" alt="image" src="https://user-images.githubusercontent.com/12021443/163877400-6a67d7c2-ace7-427d-bdb3-a926efef965b.png">

Reply:
In this case it is there to be managed imperatively by a parent directive. Let's just write `[tuiDroprown]="false"` there as proper initial value.

Reply:
fixed